data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/AwsConfig.java

```diff
@@ -10,10 +10,26 @@
 
 public class AwsConfig {
     @JsonProperty("msk_arn")
-    @Size(min = 20, max = 2048, message = "mskArn length should be between 20 and 2048 characters")
-    private String awsMskArn;
+    @Size(min = 20, max = 2048, message = "msk_arn length should be between 20 and 2048 characters")
+    private String mskArn;
 
-    public String getAwsMskArn() {
-        return awsMskArn;
+    @Size(min = 1, message = "Region cannot be empty string")
+    @JsonProperty("region")
+    private String region;
+
+    @Size(min = 20, max = 2048, message = "sts_role_arn length should be between 20 and 2048 characters")
+    @JsonProperty("sts_role_arn")
+    private String stsRoleArn;
+
+    public String getMskArn() {
```

Comment:
We will most likely need to add an option to specify what type of bootstrap servers we want to read from MSK (public, single-vpc or multi-VPC) we can probably add that later also.

Reply:
Isn't that decided on the MSK server side? Does client have option to read public if the server is not public?

Reply:
Will add this in the next PR.